unit-tests.sh

```diff
@@ -5,19 +5,21 @@ set -x
 # Note that the pytests delete nodes_log table !
 #
 
-curl localhost:5000/beehives -d '{"id": "my-beehive", "key-type": "rsa-sha2-256", "rmq-host":"host", "rmq-port": 5, "upload-host":"host", "upload-port": 6}'
-cd test-data/beehive_ca
-curl -F "tls-key=@tls/cakey.pem" -F "tls-cert=@tls/cacert.pem"  -F "ssh-key=@ssh/ca" -F "ssh-pub=@ssh/ca.pub" -F "ssh-cert=@ssh/ca-cert.pub"  localhost:5000/beehives/my-beehive
+curl -s localhost:5000/beehives -d '{"id": "my-beehive", "key-type": "rsa-sha2-256", "rmq-host":"host", "rmq-port": 5, "upload-host":"host", "upload-port": 6}'
 
+# TODO(sean) If we're going to use test keys which are valid forever, we may as well hard code them as test
+# data. Then, we can just init new beehives with a function.
+(
+cd test-data/beehive_ca
+curl -s -F "tls-key=@tls/cakey.pem" -F "tls-cert=@tls/cacert.pem"  -F "ssh-key=@ssh/ca" -F "ssh-pub=@ssh/ca.pub" -F "ssh-cert=@ssh/ca-cert.pub"  localhost:5000/beehives/my-beehive
+)
 
-until docker exec -i beekeeper_bk-sshd_1 test -e /home_dirs/node-0000000000000001/rtun.sock
-do
+# NOTE(sean) --no-TTY allows docker-compose exec to run correctly in CI.
+until docker-compose exec --no-TTY bk-sshd test -e /home_dirs/node-0000000000000001/rtun.sock; do
   echo waiting for /home_dirs/node-0000000000000001/rtun.sock
   sleep 1
 done
 
 ### TEST BEEKEEPER API
 set -x
-docker exec beekeeper_bk-api_1 /bin/bash -c 'coverage run -m pytest -v  &&  coverage report -m --fail-under 85 --include=./*'
-
-
+docker-compose exec --no-TTY bk-api /bin/bash -c 'coverage run -m pytest -v &&  coverage report -m --fail-under 85 --include=./*'
```

Comment:
i always wanted this `fail-under` number to be higher.. but had to fight Wolfgang on it :p

Reply:
I was expecting to see the results of this coverage test... see which lines were covered and which were not?  any chance we could not hide that output?

i thought I remember `pytest` giving you a full print out of the files tested and the lines not tested in each or something?

Reply:
Ah, I may have accidentally "hidden" that when I added --no-TTY.

I (may have mistakenly) thought that was causing another problem, but will revise it to see.

Reply:
Fixed now. I'm getting the coverage report again.